lerna.json

```diff
@@ -5,8 +5,8 @@
   "version": "independent",
   "command": {
     "publish": {
-      "allowBranch": "master",
-      "message": "chore(release): publish :tada:",
+      "allowBranch": "automation/lerna",
+      "message": "chore(release): Publish updated package versions to NPM :tada:",
```

Comment:
nit: This commit isn't so much about publishing direct to NPM; it's more about ensuring the package.json and CHANGELOG file(s) are updated so that when the "Publish from package.json" Github Action workflow is manually run, Lerna publishes whatever is currently on `master` at the version(s) specified in the package.json file(s) for each package.